codemeta.json

```diff
@@ -0,0 +1,296 @@
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/robparrish"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Lori",
+            "additionalName": "A.",
+            "familyName": "Burns",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0003-2852-5864",
+            "@id": "https://github.com/loriab"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Daniel",
+            "additionalName": "G. A.",
+            "familyName": "Smith",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0001-8626-0900",
+            "@id": "https://github.com/dgasmith"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Andrew",
+            "additionalName": "C.",
+            "familyName": "Simmonett",
+            "affiliation": "National Institutes of Health -- National Heart, Lung and Blood Institute",
+            "@id": "https://orcid.org/0000-0002-5921-9272",
+            "@id": "https://github.com/andysim"
+        },
+        {
+            "@type": "Person",
+            "givenName": "A.",
+            "additionalName": "Eugene",
+            "familyName": "DePrince",
+            "affiliation": "",
+            "affiliation": "Department of Chemistry and Biochemistry, Florida State University",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/edeprince3"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Edward",
+            "additionalName": "G.",
+            "familyName": "Hohenstein",
+            "affiliation": "Department of Chemistry and Biochemistry, The City College of New York",
+            "@id": "https://orcid.org/0000-0002-2119-2959",
+            "@id": "https://github.com/????"
+        },
+        {
+            "@type": "Person",
+            "givenName": "U\u0223ur",
+            "familyName": "Bozkaya",
+            "affiliation": "Department of Chemistry, Hacettepe University, Turkey",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Alexander",
+            "additionalName": "Yu.",
+            "familyName": "Sokolov",
+            "affiliation": "Division of Chemistry and Chemical Engineering, California Institute of Technology",
+            "@id": "https://orcid.org/0000-",
+            "@id": "https://github.com/ssh2"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Roberto",
+            "familyName": "Di Remigio",
+            "affiliation": "Centre for Theoretical and Computational Chemistry, UiT, The Arctic University of Norway",
```

Comment:
Can the affiliation be changed to "Hylleraas Centre for Quantum Molecular Sciences, UiT - The Arctic University of Norway"?